convene-web/app/javascript/src/utilities/card.scss

```diff
@@ -0,0 +1,12 @@
+@layer components {
+  .card {
+    @apply grid grid-cols-1 grid-rows-2 grid-flow-col shadow rounded-lg px-3 bg-white;
+  }
+  .card__header {
+    @apply w-full items-center;
+  }
+
+  .card__footer {
+    @apply w-full flex flex-col items-center justify-center border-t border-neutral-200;
+  }
```

Comment:
For utilities, I'm thinking we'll probably want to go "real BEM" instead of "Composable BEM" because we can't `@apply .card .header` to mix in the utilities.

Reply:
I like this!